### PR TITLE
ADEN-8292 Fix Krux

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: src/ad-services/krux/index.js [hunk 1]
```diff
@@ -82,12 +82,13 @@ class Krux {
...
-		const user = getKruxData('kxuser');
+		const oldUser = getKruxData('kxuser');
+		const newUser = getKruxData('kxwikia_user');
 		const segments = getKruxData('kxsegs');
 
-		context.set('targeting.kuid', user || null);
+		context.set('targeting.kuid', oldUser || newUser || null);
 		context.set('targeting.ksg', segments ? segments.split(',') : []);
-		utils.logger(logGroup, 'data set', user, segments);
+		utils.logger(logGroup, 'data set', oldUser, segments);
```
Comment (jacekbj) [inline src/ad-services/krux/index.js]: Shouldn't that be `utils.logger(logGroup, 'data set', oldUser, segments);` -> `utils.logger(logGroup, 'data set', oldUser || newUser, segments);`?

File: src/ad-services/krux/index.js [hunk 2]
```diff
@@ -82,12 +82,13 @@ class Krux {
 	 * @returns {void}
 	 */
 	importUserData() {
-		const user = getKruxData('kxuser');
+		const oldUser = getKruxData('kxuser');
```
Comment (fraszczakszymon) [inline src/ad-services/krux/index.js]: What about 
```suggestion
		const user = getKruxData('kxuser') || getKruxData('kxwikia_user');
```

File: src/ad-services/krux/index.js [hunk 3]
```diff
@@ -82,12 +82,13 @@ class Krux {
...
+		const newUser = getKruxData('kxwikia_user');
 		const segments = getKruxData('kxsegs');
```
Comment (fraszczakszymon) [inline src/ad-services/krux/index.js]: Should we handle segments too?
```suggestion
		const segments = getKruxData('kxsegs') || getKruxData('kxwikia_segs');
```

File: src/ad-services/krux/index.js [hunk 4]
```diff
@@ -82,8 +82,8 @@ class Krux {
 	 * @returns {void}
 	 */
 	importUserData() {
-		const user = getKruxData('kxuser');
-		const segments = getKruxData('kxsegs');
+		const user = getKruxData('kxuser') || getKruxData('kxwikia_user');
```
Comment (fraszczakszymon) [inline src/ad-services/krux/index.js]: Just wonder if we shouldn't make it configurable in context (e.g. `services.krux.userKey`) and use it as a fallback 
```suggestion
		const user = getKruxData('kxuser') || getKruxData(context.get('services.krux.userKey'));
```

**Pull request merged**